### PR TITLE
Integration changes 

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

File: docs/integration/index.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ Announcements are supported with the following features:
...
 - If a player type has native support for announcements (e.g. Sonos) then that will be used in preference to the MA functionality providing an even smoother experience
 - Works with all players that are supported by Music Assistant
 - For TTS (Text to Speech) announcements, support is provided for prepending it with a "pre announcement sound". This sound can be enabled/disabled in the HA integration options
 - Each player has settings which allow for adjustment of certain aspects of the announcement playback
 
 With some additional setup voice control of MA via HA is also possible.
 
-Music Assistant was originally designed to solely work with HA and as a result this tight connection has been maintained and installing the integration will also install the server as an addon. This provides a minimal effort and most secure way of getting up and running. However, this is not the only way and if you have installed the MA server elsewhere (or as an addon before) you will be able to configure the integration to connect to it.
+The integration can connect to the MA server which is running as a HA addon or if it is running separately in a docker container.
```
Comment (jozefKruszynski) [inline docs/integration/index.md:26]:
```suggestion
The integration can connect to the MA server which is running either as an HA addon or as a docker container on the same or another host system.
```

**Review by jozefKruszynski — CHANGES_REQUESTED** [review]
Possible change

File: docs/integration/installation.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
-
 !!! note 
-    The HA integration will create new media_player entities for those player types which are supported natively by MA. To see the names of those players go to `HA settings>>Devices & services>>Integrations>>Music Assistant` and view the entities. It is these players that need to be targeted in your automations.
+    The HA integration will create new media_player entities for those player types which are supported natively by MA. To see the names of those players go to `HA settings>>Devices & services>>Integrations>>Music Assistant` and view the entities. It is these players that need to be targeted in automations.
 
 ## Installation of the Home Assistant Player Provider
 
-Once the HA Integration is installed it is possible to stream to HA media player entities. In order to do that you need to install the HA Player Provider.  First you must install the Home Assistant Plug-in provider
+Once the HA Integration is installed it is possible to stream to HA media player entities. In order to do that the HA Player Provider need to be installed.  However, first the Home Assistant Plug-in provider needs to be installed.
```
Comment (jozefKruszynski) [inline docs/integration/installation.md:21]:
```suggestion
Once the HA Integration is installed it is possible to stream to HA media player entities. In order to do that the HA Player Provider needs to be installed.  However, first the Home Assistant Plug-in provider needs to be installed.
```

File: docs/music-providers/filesystem.md [hunk 3]
```diff
@@ -45,11 +45,11 @@ Music Assistant has support for SMB (also known as samba or CIFS) shares and DFS
...
-- MA prefers an Album Artist to be set for all albums. There is a setting under Advanced in the provider configuration for what to do if a track is found without this set. The default is to skip the track, so if your tracks are not being imported into the Library then check for this (there will be warnings in the log)
   
 ## Tagging Files 
 
 - It is very important that all of your audio files contain proper [ID3 tag](https://en.wikipedia.org/wiki/ID3) information. The more comprehensive the tagging the better the results will be when using MA. For this reason it is strongly recommended that all files are tagged with [MusicBrainz Picard](https://picard.musicbrainz.org). This will ensure consistency and completeness of the tags that MA needs to work best. Other programs such as [Mp3Tag](https://www.mp3tag.de/en/) are often also based on the Musicbrainz catalog and can work as well provided they include ISRC and all MBID tags.
+- MA requires the Album Artist tag to be set. If you do not have that tag set then what happens to those tracks when the provider is scanned depends on the `Action when a track is missing the Albumartist ID3 tag` setting for the local provider
```
Comment (jozefKruszynski) [inline docs/music-providers/filesystem.md:52]: Should Albumartist have a space in this sentence `Action when a track is missing the Albumartist ID3 tag`? I don't actually know....

File: docs/player-support/airplay.md [hunk 4]
```diff
@@ -12,12 +12,12 @@ Music Assistant has support for Airplay based devices. This includes Apple devic
...
 - To enable playback to a Macbook, you need to enable access to "everyone on the same network" in the Airplay settings of the Macbook
 - Because Apple TV's require authentication, they are not supported yet (but will be in the future if there's any demand)
-- Samsung seems to have implemented AirPlay 2 in a way that it isn't fully backwards compatible. Everything seems to work, changing volume, song info is shown, and you can control the Samsung device as expected, however there is no sound. Users of similar applications such as Roon and anything based on slimproto have the same problem. 
-- Some devices (such as Kodi or some 3rd party airplay receivers) require encryption. You can enable encryption in the advanced player settings if there is no sound
+- Samsung seems to have implemented AirPlay 2 in a way that it isn't fully backwards compatible. Everything seems to work, changing volume, song info is shown, and you can control the Samsung device as expected, however there is no sound. Users of similar applications such as Roon and anything based on slimproto have the same problem
```
Comment (jozefKruszynski) [inline docs/player-support/airplay.md:19]: Rather than removing the . here, should we not actually add one at the end of each of these sentences?

**Review by jozefKruszynski — CHANGES_REQUESTED** [review]
A couple more suggestions and questions

File: docs/music-providers/filesystem.md (hunk 3 above)
Comment (OzGav) [reply docs/music-providers/filesystem.md:52]: In this case I have used the actual tag name

Comment (jozefKruszynski) [reply docs/music-providers/filesystem.md:52]: I thought that might be the case, just wanted to double check

File: docs/player-support/airplay.md (hunk 4 above)
Comment (OzGav) [reply docs/player-support/airplay.md:19]: I decided to fullstop paragraphs but not the dot points and have been slowly standardising that as I update pages. I think I started when the fullstop caused the dot point to spill to another line…?

File: docs/music-providers/filesystem.md (hunk 3 above)
Comment (OzGav) [reply docs/music-providers/filesystem.md:52]: Actually I take that back. This is the exact title of the setting in the UI

File: docs/player-support/airplay.md (hunk 4 above)
Comment (jozefKruszynski) [reply docs/player-support/airplay.md:19]: fair enough

**Review by jozefKruszynski — APPROVED** [review]

**Pull request merged**